pkg/config/application.go

```diff
@@ -23,19 +24,46 @@ type Application struct {
 	ACLs          map[string]string
 }
 
-// AddApplicationOrg adds an organization to an existing config's Application configuration.
-// Will not error if organization already exists.
-func AddApplicationOrg(config *cb.Config, org Organization) error {
-	appGroup := config.ChannelGroup.Groups[ApplicationGroupKey]
+// GetApplicationConfiguration returns the existing application configuration values from a config
+// transaction as an Application type. This can be used to retrieve existing values for the application
+// prior to updating the application configuration.
+func (c *ConfigTx) GetApplicationConfiguration() (Application, error) {
```

Comment:
why are we not returning `*Application` (i mean the pointer to the Appplication)? Any specific reason?

Reply:
There's no need ... Application is simply a struct with no member functions.